doc: improve reload page in watch-files.mdx (#3318)
web-infra-dev/rsbuild: website/docs/en/config/dev/watch-files.mdx

### reload-page

> `reload-page` means that the browser page will automatically reload when a file changes. If you don't explicitly specify a type, the system will default to the reload-page behavior.

Reviewer: ```suggestion
`reload-page` means that when a file changes, the page opened in the browser will automatically reload. If the type is not explicitly specified, Rsbuild will default to the `reload-page` behavior.
```
